Core.UnitTests/CodeGeneration/ReflectionEmit/ModuleBuilderFactoryTest.cs

```diff
@@ -47,6 +47,9 @@ public void SetUp ()
       _currentDirectory = Environment.CurrentDirectory;
     }
     [Test]
+#if !FEATURE_ASSEMBLYBUILDER_SAVE
+    [Ignore]
```

Comment:
It think it sould be possible to update the tests by only changing the CheckAdapterAndSaveToDiskBehavior method to remove the failing parts. It might also be a good idea to just split up the Check-method into two parts with dedicated names and then not do the SaveToDiskBehavior calls in the tests. That way, one can see from the name of the called check methods what parts are tested and what are not tested.